templates/index.html

```diff
@@ -48,11 +48,19 @@ <h2>Test Setup Corn</h2>
     <section class="form-container" id="reservation-form">
         <h2>Make a reservation</h2>
         <form method="post" role="form">
+            {% for field, errors in form.errors.items() %}
+            <div class="form-error">
+                {{ ', '.join(errors)}}
+            </div>
+            {% endfor %}
+
             {{ form.hidden_tag() }}
             {{ form.name.label }}
             {{ form.name }}
+            <small>*{{ form.name.description }}</small>
```

Comment:
The asterisk seems unnecessary. If it's not, add it to the description, instead of to this element

Reply:
Removed asterisks